From 27c60dbf0f5c2ae76fa3fae263e9ec3f389278e8 Mon Sep 17 00:00:00 2001
From: Meir Levi <mlevi4@marvell.com>
Date: Thu, 24 Dec 2020 05:00:20 -0500
Subject: [PATCH 1049/1921] arm64: defconfig: increase CMA_SIZE_MBYTES to 256MB

Update the CMA size in mega bytes from 32 to 256 for
marvell_v8_sdk_defconfig.

Signed-off-by: Meir Levi <mlevi4@marvell.com>
Change-Id: I5c36d0c7164cca88ed7bfc3035f696566f060618
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/42760
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/configs/marvell_v8_sdk_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/marvell_v8_sdk_defconfig b/arch/arm64/configs/marvell_v8_sdk_defconfig
index 46fc575add08..17c10b485810 100644
--- a/arch/arm64/configs/marvell_v8_sdk_defconfig
+++ b/arch/arm64/configs/marvell_v8_sdk_defconfig
@@ -836,7 +836,7 @@ CONFIG_NLS_ISO8859_1=y
 CONFIG_SECURITY=y
 CONFIG_CRYPTO_ECHAINIV=y
 CONFIG_CRYPTO_ANSI_CPRNG=y
-CONFIG_CMA_SIZE_MBYTES=32
+CONFIG_CMA_SIZE_MBYTES=256
 CONFIG_PRINTK_TIME=y
 CONFIG_DEBUG_INFO=y
 CONFIG_DEBUG_FS=y
-- 
2.31.1

